### ISSUE TITLE
gradle build failed

### ISSUE THREAD

**Issue by sangshuduo** [dist-sys/mqttloader#17]
$ gradle build

FAILURE: Build failed with an exception.

\* Where:

Build file '/home/shuduo/work/mqtt/mqttloader/build.gradle' line: 25

\* What went wrong:

A problem occurred evaluating root project 'mqttloader'.

\> Could not set unknown property 'release' for object of type org.gradle.api.tasks.compile.CompileOptions.

\* Try:

Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

\* Get more help at https://help.gradle.org

BUILD FAILED in 0s

**Comment by sangshuduo** [comment]
$ gradle --version

\------------------------------------------------------------

Gradle 4.4.1

\------------------------------------------------------------

Build time:   2012-12-21 00:00:00 UTC

Revision:     none

Groovy:       2.4.16

Ant:          Apache Ant(TM) version 1.10.5 compiled on March 28 2019

JVM:          1.8.0\_265 (Private Build 25.265-b01)

OS:           Linux 5.4.0-52-generic amd64

$ java -version

openjdk version "1.8.0\_265"

OpenJDK Runtime Environment (build 1.8.0\_265-8u265-b01-0ubuntu2~18.04-b01)

OpenJDK 64-Bit Server VM (build 25.265-b01, mixed mode)

$ cat /etc/issue

Ubuntu 18.04.5 LTS \\n \\l

**Comment by nabon** [comment]
Thank you for your valuable comment, and sorry for the late reply.
The release flag option (options.release) was added as one of the features of Gradle 6.6.
I set this option to ensure the compatibility with earlier version of Java.

I think the possible solutions for the issue of gradle build failure are:
- Remove the release flag option from build.gradle. (Thank you for the PR #18)
- Use the newer version of Gradle (6.6 or later), while keeping the build.gradle as it is.

The former is developer-friendly (because they can use older versions of gradle), while the latter is user-friendly (because they can use older versions of Java).

 I'd like to keep the current build.gradle file for the user-friendliness.
